libs/net/dal/Services/ReportService.cs

```diff
@@ -899,6 +899,42 @@ public IEnumerable<ReportInstance> GetLatestInstances(int id, int? ownerId = nul
                 searchResults.Add(section.Name, content);
                 excludeAboveSectionContentIds.AddRange(contentIds.Except(excludeAboveSectionContentIds));
             }
+            else if (!reportSettings.Content.ClearOnStartNewReport
```

Comment:
This condition wasn't considered in earlier iterations.  When a section doesn't have a datasource it needs to keep the prior data when configured to accumulate.